L1Trigger/L1TGlobal/plugins/L1TGlobalProducer.cc

```diff
@@ -403,6 +401,17 @@ void L1TGlobalProducer::produce(edm::Event& iEvent, const edm::EventSetup& evSet
       m_l1GtMenu->print(std::cout, printV);
 
     m_l1GtMenuCacheID = l1GtMenuCacheID;
+
+    //for getting model version to condition class via GlobalBoard.runGTL, comes from menu rather than config
+    if (gtParser.gtTriggerMenuName() == "L1Menu_Collisions2024_v1_0_0") {
+      m_AXOL1TLModelVersion = "";
+
+      edm::LogWarning("L1TGlobalProducer")
+          << "Warning: menu version " << gtParser.gtTriggerMenuName()
+          << " is incompatible with AXOL1TL, will not evaluate conditions" << std::endl;
```

Comment:
...and remove this LogWarning?

Reply:
this has been fixed in most recent commit: https://github.com/cms-sw/cmssw/commit/26756ff6882c227e078d9ed2c20bf0317243238b